Assets/Scripts/Player/CameraFollow.cs

```diff
@@ -85,14 +86,6 @@ public void Update()
         if (Input.GetButton("Camera Vertical Rotation"))
             vAngleDelta = Input.GetAxis("Camera Vertical Rotation") * VERTICAL_ROTATION_SENSITIVITY * Time.deltaTime;
 
-        // Camera mouse movement: On right click being held down
-        if (Input.GetMouseButton(1)) // There isnt a fucking enum for the mouse buttons
-        {
-            // Use mouse movement to determine axes
-            angleDelta = Input.GetAxis("Mouse X");
-            vAngleDelta = -Input.GetAxis("Mouse Y");
-        }
```

Comment:
What's the removal for?

Reply:
Since we have public builds, I thought it would be best to have the camera be as it will in proper play mode, IE strict top down, rotations with Q/E and zoom with mouse wheel.

Reply:
Huh, i always use the right click for rotating around, RTS style

Reply:
Yes, but that is not how Beep* envisioned it, nor how we have it planned, afaik. Since we are slowly adding gameplay, I thought we should begin using the actual player camera, so we can see what works and doesn't work as we add features.

Reply:
That makes sense. If so i need to do some more investigation into the view angles & stuff cos something doesn't quite feel right to me. Still, allg by me then for now, though probably better to have done this in a separate PR.